From 234df47bc5f6db1f7317deb36507ec2dce598306 Mon Sep 17 00:00:00 2001
From: Cliff Wu <z800201002005@gmail.com>
Date: Fri, 4 Aug 2023 11:38:47 +0900
Subject: [PATCH 1/2] Backport from d57c994 to fix compilation failure

---
 gcc/reload.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/reload.h b/gcc/reload.h
index 65fa29c2bd2..e3ec30ac858 100644
--- a/gcc/reload.h
+++ b/gcc/reload.h
@@ -166,7 +166,7 @@ struct target_reload {
      value indicates the level of indirect addressing supported, e.g., two
      means that (MEM (MEM (REG n))) is also valid if (REG n) does not get
      a hard register.  */
-  bool x_spill_indirect_levels;
+  unsigned char x_spill_indirect_levels;
 
   /* True if caller-save has been reinitialized.  */
   bool x_caller_save_initialized_p;
-- 
2.39.2

